src/TTrigger.cpp

```diff
@@ -171,7 +171,7 @@ bool TTrigger::setRegexCodeList(QStringList regexList, QList<int> propertyList)
 
             int erroffset;
 
-            QSharedPointer<pcre> re(pcre_compile(local8Bit.constData(), 0, &error, &erroffset, 0), pcre_deleter);
+            QSharedPointer<pcre> re(pcre_compile(local8Bit.constData(), 0, &error, &erroffset, nullptr), pcre_deleter);
```

Comment:
More 'C' library code that may want the `NULL` `#define` rather than `0`/`nullptr`.